pkg/sdk/testint/databases_integration_test.go

```diff
@@ -80,248 +94,712 @@ func TestInt_DatabasesCreate(t *testing.T) {
+
+	externalDatabaseId := sdk.NewExternalObjectIdentifier(secondaryTestClientHelper().Ids.AccountIdentifierWithLocator(), sharedDatabase.ID())
+	comment := random.Comment()
+	err = client.Databases.CreateSecondary(ctx, sharedDatabase.ID(), externalDatabaseId, &sdk.CreateSecondaryDatabaseOptions{
+		IfNotExists:                sdk.Bool(true),
+		DataRetentionTimeInDays:    sdk.Int(1),
+		MaxDataExtensionTimeInDays: sdk.Int(10),
+		ExternalVolume:             &externalVolume,
+		Catalog:                    &catalog,
+		DefaultDDLCollation:        sdk.String("en_US"),
+		LogLevel:                   sdk.Pointer(sdk.LogLevelDebug),
+		TraceLevel:                 sdk.Pointer(sdk.TraceLevelAlways),
+		Comment:                    sdk.String(comment),
+	})
+	require.NoError(t, err)
+	t.Cleanup(func() {
+		err = client.Databases.Drop(ctx, sharedDatabase.ID(), nil)
 		require.NoError(t, err)
-		assert.Equal(t, databaseID.Name(), database.Name)
-	}
-*/
-func TestInt_DatabasesDescribe(t *testing.T) {
-	client := testClient(t)
+	})
 
-	// new database and schema created on purpose
-	databaseTest, databaseCleanup := testClientHelper().Database.CreateDatabase(t)
-	t.Cleanup(databaseCleanup)
-	schemaTest, schemaCleanup := testClientHelper().Schema.CreateSchemaInDatabase(t, databaseTest.ID())
-	t.Cleanup(schemaCleanup)
-	ctx := testContext(t)
-	databaseDetails, err := client.Databases.Describe(ctx, databaseTest.ID())
+	database, err := client.Databases.ShowByID(ctx, sharedDatabase.ID())
 	require.NoError(t, err)
-	rows := databaseDetails.Rows
-	found := false
-	for _, row := range rows {
-		if row.Name == schemaTest.ID().Name() {
-			found = true
-		}
-	}
-	assert.True(t, found)
+
+	assert.Equal(t, sharedDatabase.ID().Name(), database.Name)
+	assert.Equal(t, 1, database.RetentionTime)
+	assert.Equal(t, comment, database.Comment)
+
+	param, err := client.Parameters.ShowObjectParameter(ctx, "MAX_DATA_EXTENSION_TIME_IN_DAYS", sdk.Object{ObjectType: sdk.ObjectTypeDatabase, Name: sharedDatabase.ID()})
+	assert.NoError(t, err)
+	assert.Equal(t, "10", param.Value)
+
+	externalVolumeParam, err := client.Parameters.ShowObjectParameter(ctx, "EXTERNAL_VOLUME", sdk.Object{ObjectType: sdk.ObjectTypeDatabase, Name: sharedDatabase.ID()})
+	assert.NoError(t, err)
+	assert.Equal(t, externalVolume.Name(), externalVolumeParam.Value)
+
+	catalogParam, err := client.Parameters.ShowObjectParameter(ctx, "CATALOG", sdk.Object{ObjectType: sdk.ObjectTypeDatabase, Name: sharedDatabase.ID()})
+	assert.NoError(t, err)
+	assert.Equal(t, catalog.Name(), catalogParam.Value)
+
+	logLevelParam, err := client.Parameters.ShowObjectParameter(ctx, "LOG_LEVEL", sdk.Object{ObjectType: sdk.ObjectTypeDatabase, Name: sharedDatabase.ID()})
+	assert.NoError(t, err)
+	assert.Equal(t, string(sdk.LogLevelDebug), logLevelParam.Value)
+
+	traceLevelParam, err := client.Parameters.ShowObjectParameter(ctx, "TRACE_LEVEL", sdk.Object{ObjectType: sdk.ObjectTypeDatabase, Name: sharedDatabase.ID()})
```

Comment:
same as above, some asserts can be extracted

Reply:
changed, the common assert would have to be extracted to be top-level, so I left it as is.